### PR TITLE
Add container image and scitokens-cpp discussion

### PR DESCRIPTION
**Pull request by brianhlin**
@jthiltges @djw8605 did I miss anything in the notes for our discussion?

### CODE REVIEW

File: docs/meetings/2023/TechArea20231107.md [hunk 1]
```diff
@@ -39,15 +39,31 @@ Remaining OSG 23 items <https://opensciencegrid.atlassian.net/browse/SOFTWARE-55
...
+    -   For the time being for the `xrootd-monitoring-collector`, we should just use a custom GitHub Action
+-   Release of scitokens-cpp is in bodhi and needs karma. Ready for TimT tobuild for Debian
+    -   EPEL7: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2023-f0f162d810
+    -   EPEL8: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2023-f9dc18f284
+    -   EPEL9: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2023-2dfb911a54
 
 ### Support Update
 
-None this week
+-   cvmfs-x509-helper issue with the latest version that causes the client to hang. DaveD is on it.
```
Comment (djw8605) [inline docs/meetings/2023/TechArea20231107.md:66]: I think it's a CVMFS issue, not a cvmfs-x509-helper issue.

**Review by djw8605 — DISMISSED** [review]
1 comment

File: docs/meetings/2023/TechArea20231107.md (hunk 1 above)
Comment (brianhlin) [reply docs/meetings/2023/TechArea20231107.md]: Fixed, thanks!

File: docs/meetings/2023/TechArea20231107.md [hunk 2]
```diff
@@ -39,15 +39,31 @@ Remaining OSG 23 items <https://opensciencegrid.atlassian.net/browse/SOFTWARE-55
...
+        well as immutability/retention policies
+    -   Repo sprawl is a concern for security and management.
+        Harbor creds are per organization so it may be prudent to separate out mostly-internally facing containers to a
+        separate project, e.g. `osg-internal`
+    -   Consolidation into the `images` repo doesn't work well for many images right now, especially those with "large"
+        code bases as it makes dev (think tracking issues/PRs) more difficult.
+        We should think of the SVN packaging repo as an analog here.
+        We could add some additional flexibility to the `images` repo that would allow per-image references to external,
+        trusted repos and usage of build processes that aren't so tightly tied to the Software release process.
+    -   For the time being for the `xrootd-monitoring-collector`, we should just use a custom GitHub Action
```
Comment (jthiltges) [inline docs/meetings/2023/TechArea20231107.md:58]:
```suggestion
    -   For the time being for the `xrootd-monitoring-collector`, we should just use a custom GitHub Action
    -   Implementation discussion in [SOFTWARE-5013](https://opensciencegrid.atlassian.net/browse/SOFTWARE-5013)
```

**Review by jthiltges — APPROVED** [review]
LGTM. Maybe add link to jira issue.

**Pull request merged**